run-nitro-enclaves-acm-tests

```diff
@@ -273,7 +287,7 @@ function test_token_ec_secp384r1_key {
 
 	out=$($P11NE_CLI describe-device)
 	ttl_init=$($P11NE_CLI describe-device \
-		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token")' \
+		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token-secp384r1-ec")' \
```

Comment:
Same question, why was this change necessary?

Reply:
The same explanation as in the previous case applies here.